STEER/STEERBase/AliGenHepMCEventHeader.h

```diff
@@ -95,6 +105,12 @@ class AliGenHepMCEventHeader : public AliGenEventHeader
    Double_t fpdf1;       // PDF (id1, x1, Q) - x*f(x)
    Double_t fpdf2;       // PDF (id2, x2, Q) - x*f(x)
 
+   // For pp generators
+   Double_t  fSigmaGen;   ///< Cross section
+   Double_t  fSigmaErr;   ///< Error on the cross section
+   Double_t  fPtHard;     ///< Pt of the hard process
+   Int_t     fNtrials;    ///< Number of trials
+
    ClassDef(AliGenHepMCEventHeader, 2)  // Event header for HepMC event
```

Comment:
I would increment the classDef version

Reply:
@preghenella Thanks for the reminder! Done